src/components/InfoPanel/InfoPanel.js

```diff
@@ -66,7 +66,7 @@ const InfoPanel = () => {
       {showExpandedTopology ? (
         <Modal
           close={() => setShowExpandedTopology(false)}
-          title={modelName.split("/")[1] || "Error: model not loaded..."}
+          title={modelName.split("/")[1] || modelName}
```

Comment:
You'll still want to keep the old fallback as a third option as that was added to resolve another issue.

Reply:
The original issue was preventing a blank value if the model name could not be split on slash... this change addresses that already.